frontend/tests/fixtures.ts

```diff
@@ -61,4 +69,46 @@ export const test = base.extend<Fixtures>({
     await deleteUser(context.request, tempUser.id);
     await context.close();
   },
+  tempProject: async ({ page }, use, testInfo) => {
+    const titleForCode =
+      testInfo.title
+      .replaceAll(' ','-')
+      .replaceAll(/[^a-z-]/g,'');
+    const code = `test-${titleForCode}-${testInfo.testId}`;
```

Comment:
I didn't dive into the Playwright code to see how it's generated, but it's a string containing only lowercase alphanumerics and hyphens. It's clearly hash-based in some way: it's always the same ID for the same test on the same browser, but a different ID for the same test running in Chromium vs. Firefox. This makes it a good choice for this test because the Chromium and Firefox versions of the test can run in parallel without interfering with each other, but different runs of the test will generate the same project code each time (so if in the future we decide to use screenshots, we can).